### PR TITLE
Update weapons.dm

### PR DESCRIPTION
**Pull request by icantthinkofanameritenow**
Switching AEG design to AEG modkit design

### CODE REVIEW

**Comment by Shadowmech88** [comment]
What you currently have as the description would be better suited as the title.

Additionally, PRs don't need to be THIS atomic. Both this commit and the one in #8343 could be combined into a single "Makes AEGs Obtainable Via Modkit Instead of Protolathe" PR.

File: code/modules/research/designs/weapons.dm [hunk 1]
```diff
@@ -1,13 +1,13 @@
 /datum/design/nuclear_gun
-	name = "Advanced Energy Gun"
-	desc = "An energy gun with an experimental miniaturized reactor."
+	name = "Advanced Energy Gun Modkit"
+	desc = "Can be used on an energy gun to grant it an experimental miniaturized reactor."
 	id = "nuclear_gun"
 	req_tech = list("combat" = 3, "materials" = 5, "powerstorage" = 3)
 	build_type = PROTOLATHE
-	materials = list(MAT_IRON = 5000, MAT_GLASS = 1000, MAT_URANIUM = 500)
+	materials = list(MAT_IRON = 5000, MAT_GLASS = 1500, MAT_URANIUM = 750)
```
Comment (ghost) [inline code/modules/research/designs/weapons.dm:7]: I would assume it'd cost less, not more, to make a modkit, considering you're using parts from elsewhere.

**Comment by icantthinkofanameritenow** [comment]
Yeah, that's my fuckup. Hold on a moment, this is in the other PR now.

**Comment by clusterfack** [comment]
By the commit message, I assume you are using github to do this. Come on IRC so we can teach you a git client which is infinitely easier, or read the readme.

**Pull request closed without merging**